dsfdr/dsfdr.py

```diff
@@ -11,7 +11,7 @@
 
 
 # new fdr method
-def dsfdr(data, labels, transform_type='rankdata', method='meandiff',
```

Comment:
need to fix the docstring (which describes the option as "rankdata" and not "rank")

Reply:
I just fixed and updated the file. Thanks!